include/swift/SIL/SILLinkage.h

```diff
@@ -38,33 +38,51 @@ enum class SILLinkage : uint8_t {
   /// This object definition is visible to multiple Swift modules (and
   /// thus potentially across linkage-unit boundaries).  There are no
   /// other object definitions with this name in the program.
+  ///
+  /// Public functions must be definitions, i.e. must have a body, except they
+  /// are clang imported.
```

Comment:
Nitpick -- s/clang imported/body emitted by clang/? This is only true in a few narrow cases like bitfield accessors and static inline functions IIRC